internal/pkg/describe/rd_web_service.go

```diff
@@ -238,11 +246,11 @@ func (w *rdWebSvcDesc) HumanString() string {
 	}
 	fmt.Fprint(writer, color.Bold.Sprint("\nRoutes\n\n"))
 	writer.Flush()
-	headers := []string{"Environment", "URL"}
+	headers := []string{"Environment", "Public", "URL"}
```

Comment:
Samesies here, what do you think of "Ingress" as the column name with the values ("public" or "private (environment)") for example